test/e2e_new/multiple_identities_test.go

```diff
@@ -176,4 +176,55 @@ var _ = Describe("[PR] When deploying multiple identities", func() {
 		}
 		Expect(time.Since(start) <= 150*time.Second).To(BeTrue(), "Creation and validation of 40 AzureAssignedIdentities took more than 150 seconds")
 	})
+
+	It("should create a new AzureAssignedIdentity when the pod label is changed", func() {
```

Comment:
Can we enable this as part of the PR?

Reply:
It's already running as part of the PR: https://dev.azure.com/azure/aad-pod-identity/_build/results?buildId=12013&view=logs&j=011e1ec8-6569-5e69-4f06-baf193d1351e&t=e5bbc628-0816-5281-89e5-f3e62109ab4a&l=858